suricata.yaml.in

```diff
@@ -1702,6 +1702,11 @@ detect:
   inspection-recursion-limit: 3000
   # maximum number of times a tx will get logged for a stream-only rule match
   # stream-tx-log-limit: 4
+  # try to tie an app-layer transaction for rules without app-layer keywords
+  # if there is only one live transaction for the flow
+  # allows to log app-layer metadata in alert
+  # but the transaction may not be the relevant one.
+  # force-tie-applayer-tx: no
```

Comment:
should we name this `guess-applayer-tx` to make clear it's going to be hit and miss?

Reply:
ok thanks

Reply:
Is it still a guess if there is one and only one tx?  I suppose the segment of data could be unrelated to the tx?

Reply:
> I suppose the segment of data could be unrelated to the tx

yes